tools/serve/serve.py

```diff
@@ -885,7 +885,7 @@ class ConfigBuilder(config.ConfigBuilder):
             "https-public": ["auto"],
             "ws": ["auto"],
             "wss": ["auto"],
-            "webtransport-h3": ["auto"],
+            "webtransport-h3": [11000],
```

Comment:
I think we should keep this "auto" and override it as needed in somewhere else, like environment.py or ConfigBuilder.

Reply:
Oops, sorry, I mis-uploaded my local settings.